vsintegration/src/FSharp.Editor/CodeFixes/FSharpCodeFixContext.fs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Text
+
+open CancellableTasks
+
+type FSharpCodeFixContext(document: Document, span: TextSpan) =
```

Comment:
@T-Gro This is where I see having error messages in some form as well to be used by code fixes if needed.

This will need some effort to bring it here since Roslyn and F# have different representations of diagnostics. But we have some convertors for that so shouldn't be very hard.